packages/filebrowser-extension/src/index.ts

```diff
@@ -405,6 +409,17 @@ function addCommands(app: JupyterLab, tracker: InstanceTracker<FileBrowser>, bro
     label: 'Copy Shareable Link'
   });
 
+  commands.addCommand(CommandIDs.copyPath, {
+    execute: () => {
+      const path = browser.selectedItems().next().path;
```

Comment:
Strictly speaking, I think it is possible for this command to be executed when there are not any selected items, so the first item grabbed from the iterator can be `undefined`. A slightly more defensive way of doing this would be to write
```typescript
const item = browser.selectedItems().next();
if (!item) {
  return;
}
Clipboard.copyToSystem(item.path);
```

Reply:
@ian-r-rose Good point! I made the change you suggested and pushed.